### PR TITLE
Clarification

### PR DESCRIPTION
**Pull request by way0utwest**
SELECT isn't included as DML, but most vendors and docs note that SELECT is DML. I also added minor clarification to DDL, as this page is hard to update, and this list feels incomplete in terms of items that alter data structures. Arguably there might be other items in here, so added a general disclaimer.

### CODE REVIEW

File: docs/t-sql/statements/statements.md [hunk 1]
```diff
@@ -39,13 +39,16 @@ Data Definition Language (DDL) statements defines data structures. Use these sta
 - RENAME
 - UPDATE STATISTICS
 
+There are other language elements that affect data structures as well.
```
Comment (CarlRabeler) [inline docs/t-sql/statements/statements.md:42]: Hi - I do not understand the purpose behind the addition of this sentence. Can you please elaborate or remove? 
Thanks,
Carl

Comment (way0utwest) [reply docs/t-sql/statements/statements.md:42]: I didn't have time to review the entirety of BOL to be sure nothing else applies, so I added this as a disclaimer. I can remove this if needed.

Comment (CarlRabeler) [reply docs/t-sql/statements/statements.md:42]: please remove that sentence

**Comment by PRMerger16** [comment]
@way0utwest : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**